di-ipv-queue-stub/enqueue-event/package.json

```diff
@@ -11,6 +11,6 @@
         "@types/uuid": "^9.0.2"
     },
     "dependencies": {
-        "stub-oauth-client": "git+ssh://git@github.com:govuk-one-login/stub-oauth-client#main"
+        "stub-oauth-client": "github:govuk-one-login/stub-oauth-client#main"
```

Comment:
I've not seen this scheme before - do you have a doc link?

Reply:
I didn't actually do this - I _think_ npm did it for me. Which is...weird. I'll see if I can find something about it

Reply:
The only thing I can find is this, but it's talking about slightly different: https://docs.npmjs.com/cli/v9/configuring-npm/package-json#repository

But it does seem to work when running npm install...